client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -164,6 +166,7 @@ const Settings = ( {
 				subtitle={ getSslReadableStatus( domain ) }
 				key="security"
 				isDisabled={ domain.isMoveToNewSitePending }
+				expanded={ !! shouldOpenSsl }
```

Comment:
```suggestion
				expanded={ queryParams.get( 'ssl-open' ) === 'true' }
```
On L334, there's an existing example of an accordion being expanded on page load. Let's follow the same approach here and drop the use of `useUrlQueryParam`